client/src/component/admin/AdminPage.js

```diff
@@ -1,11 +1,26 @@
+import { useAuth0 } from "@auth0/auth0-react";
+import { createContext, useEffect, useState } from "react";
 import { Outlet } from "react-router-dom";
 
+export const AdminContext = createContext(null);
+
 function AdminPage() {
+  const { getAccessTokenSilently } = useAuth0();
+  const [tokenData, setTokenData] = useState(null);
+  useEffect(() => {
+    (async () => {
+      const audience = process.env.REACT_APP_AUDIENCE;
+      const token = await getAccessTokenSilently({ audience });
+      setTokenData(token);
+    })();
+  });
```

Comment:
Do you want this to happen on every render?

Reply:
NO